cmd/mirror-main.go

```diff
@@ -473,6 +477,7 @@ func (mj *mirrorJob) monitorMirrorStatus() (errDuringMirror bool) {
 		}
 
 		if sURLs.SourceContent != nil {
+			s3mirrorTotalUploadedBytes.Add(float64(sURLs.SourceContent.Size))
```

Comment:
we should add it after its uploaded? @vadmeste

Reply:
> we should add it after its uploaded? @vadmeste

yes it is already doing that. This is monitorMirrorStatus() function.